src/components/Footer/index.js

```diff
@@ -1,35 +1,123 @@
+                Reading Room Hours:
+              </Trans>
+            </p>
+            <p className='footer-primary__text'>
+              <Trans comment='Reading Room Actual Hours'>
+                Monday-Friday<br />
+                10:00 a.m. to 5:00 p.m.<br />
+              </Trans>
+              <a className='footer-primary__link' href={
+                t({
+                  comment: 'Access Materials link',
+                  message: 'https://rockarch.org/collections/access-and-request-materials/'
+                })
+              }>
+                <Trans comment='How to Access materials message'>
+                  How to access and request materials.
+                </Trans>
+              </a>
+            </p>
+            <a className='footer-primary__link' href={
+              t({
+                comment: 'Holiday Schedule link',
+                message: 'https://rockarch.org/collections/access-and-request-materials/holiday-schedule'
+              })
+            }>
+              <Trans comment='Holiday Schedule message'>
+                See holiday schedule
+              </Trans>
+            </a>
           </div>
           <div className='footer-primary__social'>
             <SocialIcons />
             <p>
-              <a className='footer-primary__link footer-primary__policy-link' href='https://rockarch.org/about-us/accessibility/'>Accessibility Statement</a>
-              <a className='footer-primary__link footer-primary__policy-link' href='https://rockarch.org/about-us/privacy-policy/'>Privacy Policy</a>
-              <a className='footer-primary__link footer-primary__policy-link' href='https://docs.rockarch.org'>RAC Policies</a>
+              <a className='footer-primary__link footer-primary__policy-link' href={
+                t({
+                  comment: 'Accessibility Statement Link',
+                  message: 'https://rockarch.org/about-us/accessibility/'
+                })
+              }>
+                <Trans comment='Accessibility Statement message'>
+                  Accessibility Statement
+                </Trans>
+              </a>
+              <a className='footer-primary__link footer-primary__policy-link' href={
+                t({
+                  comment: 'Privacy Policy link',
+                  message: 'https://rockarch.org/about-us/privacy-policy/'
+                })
+              }>
+                <Trans comment='Privacy Policy message'>
+                  Privacy Policy
+                </Trans>
+              </a>
+              <a className='footer-primary__link footer-primary__policy-link' href={
+                t({
+                  message: 'https://docs.rockarch.org',
+                  comment: 'Footer.Primary.RAC.href'
+                })
+              }>
+                <Trans comment='Footer.Primary.RAC.message'>
+                  RAC Policies
+                </Trans>
+              </a>
```

Comment:
Do the translation wrappings create too much clutter compared to the original? Does it need cut down, and do you have a preference on how it is reduced?

Reply:
It is a lot more cluttered for sure. I guess one approach would be to create a PolicyLink component?

Reply:
I like the idea of decluttering with PrimaryLink and SecondaryLink components as part of this PR.